test/support/integration/plugins/modules/synchronize.py

```diff
@@ -145,7 +145,7 @@
     default: yes
   use_ssh_args:
     description:
-      - Use the ssh_args specified in ansible.cfg.
+      - Use the ssh_args specified in ansible.cfg. Setting this to `yes` will also make `synchronize` use `ansible_ssh_common_args`.
```

Comment:
i would just rephrase this to 'use the same options as the ssh connection plugin' as it also includes CLI and env vars.